modules/network/smb2man/libsmb2/lib/smb2-cmd-negotiate.c

```
@@ -0,0 +1,648 @@
+        memcpy(iov->buf + 8, rep->server_guid, SMB2_GUID_SIZE);
+        smb2_set_uint32(iov, 24, rep->capabilities);
+        smb2_set_uint32(iov, 28, rep->max_transact_size);
+        smb2_set_uint32(iov, 32, rep->max_read_size);
+        smb2_set_uint32(iov, 36, rep->max_write_size);
+        smb2_set_uint64(iov, 40, rep->system_time);
+        smb2_set_uint64(iov, 48, rep->server_start_time);
+
+        rep->security_buffer_offset = len + SMB2_HEADER_SIZE;
+        smb2_set_uint16(iov, 56, rep->security_buffer_offset);
+        smb2_set_uint16(iov, 58, rep->security_buffer_length);
+        rep->negotiate_context_offset = len + seclen + SMB2_HEADER_SIZE;
```

Cppcheck (2.21.0)

[error] Uninitialized variable

(legacyUninitvar)